### PR TITLE
[DQL2-25] QA remediation

### PR DESCRIPTION
**Pull request by salsa-nathan**
[DQL2-25] Moved 'Comments' tab to 2nd position.

@duttonw & @ThrawnCA - unable to add you guys as reviewers, but here's a PR for review - thanks.

### CODE REVIEW

File: ckanext/data_qld_theme/templates/package/read_base.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{% ckan_extends %}
```
Comment (duttonw) [inline ckanext/data_qld_theme/templates/package/read_base.html:1]: is this only related to comments plugin? is it at all possible for this to be the default template inside the comments plugins as i don't see anything that would relate to our theme (id's class's etc) to pin it.

Also should it have a new line at the end?

File: ckanext/data_qld_theme/templates/package/read_base.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+{% block content_primary_nav %}
+    {{ h.build_nav_icon('dataset_read', _('Dataset'), id=pkg.name) }}
+    {% if h.ytp_comments_show_comments_tab_page() %}
```
Comment (ThrawnCA) [inline ckanext/data_qld_theme/templates/package/read_base.html:5]: Where is this function defined? I can't find it in ckanext-data-qld or ckanext-data-qld-theme

**Comment by duttonw** [comment]
cicd failed
Failing scenarios:
  test/features/google_analytics.feature:8  When viewing the HTML source code of a dataset, the organisation meta data is visible
  test/features/google_analytics.feature:13  When viewing the HTML source code of a resource, the organisation meta data is visible

4 features passed, 1 failed, 0 skipped
14 scenarios passed, 2 failed, 0 skipped
54 steps passed, 2 failed, 1 skipped, 0 undefined
Took 0m37.883s

File: ckanext/data_qld_theme/templates/package/read_base.html (hunk 2 above)
Comment (duttonw) [reply ckanext/data_qld_theme/templates/package/read_base.html:5]: is in the comments plugin, my suggestion is that this injects/updates it via the plugin instead of inside the theme which now makes the theme dependant on comments to complete integration testing

**Comment by salsa-nathan** [comment]
@duttonw & @ThrawnCA - I'm going to close the PR - please disregard it. We will move this template override into `ckanext-ytp-comments`.
